server/api.js

```diff
@@ -1,9 +1,10 @@
 const express = require("express");
+const fetch = require('node-fetch');
 const router = express.Router();
+const tools = require('./routes/github');
 
-router.get("/", (req, res) => {
-    res.send({"foo": "bar"});
-});
+tools.fetchIssues("MLH-Fellowship", "click")
```

Comment:
I think unlike discord links and youtube videos, github prs/issues should not be one of the things we pre-populate. (Might be spammy because not all repos are relevant to everyone). However, it'll be a useful tool for when we set up the API endpoints for adding posts. So I think we shouldn't invoke them in api.js just yet.